algorithms/ar-bsrh/javascript/binary_Search.js

```diff
@@ -0,0 +1,34 @@
+function binary_Search(items, value) {
+	var firstIndex = 0,
+		lastIndex = items.length - 1,
+		middleIndex = Math.floor((lastIndex + firstIndex) / 2);
+
+	while (items[middleIndex] != value && firstIndex < lastIndex) {
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6275500)